configure_tls_protocol_version_and_ciphers.md

```diff
@@ -0,0 +1,18 @@
+# Configure TLS Protocol Version and Ciphers
+
+## SDK
+### How to configure TLS version
+Use `jdk.tls.client.protocols` system property to enforce TLS version. This property was introduced to JDK7 `7u95` and to JDK6 `6u121`.
+```
+$ java -Djdk.tls.client.protocols="TLSv1.2" yourApp
+```
+
+### How to configure TLS ciphers
+Use `jdk.tls.legacyAlgorithms` property of `java.security` file to enforce ciphers.
+On JDK 8 and earlier, `java.security` file is under folder `%JAVA_HOME%/lib/security`, for JDK 9, it’s  under folder `%JAVA_HOME%/conf/security`
+
+## Android
+To be done
+
```

Comment:
Can we not have this section until ready?

Reply:
deleted.